module/PowerShellEditorServices/Commands/Public/Set-ScriptExtent.ps1

```diff
@@ -34,11 +36,11 @@ function Set-ScriptExtent {
                 $Extent.StartOffset,
                 $Extent.EndOffset)
         }
-        $extentList.Add($Extent)
+
+        $extentList.Add($Extent.StartOffset, $Extent)
     }
-    # Currently this kills the pipeline because we need to keep track and edit all extents for position tracking.
-    # TODO: Consider queueing changes in a static property and adding a PassThru switch.
     end {
+        $needsIndentFix = $false
```

Comment:
Defensively assigning to avoid potentially getting a variable with the same name from a parent scope